src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -177,6 +178,11 @@ public void start() {
         .withName(name), DEFAULT_SELECTOR_THREAD_COUNT);
     workerScheduler = getWorkerScheduler(schedulersConfig.withName(name + ".requester.workers"));
 
+    redirectScheduler = schedulerService.customScheduler(schedulersConfig
```

Comment:
where is the schedulerConfig coming from? I would expect it to have min=0, max=selectcount and maxIdle=0. Why reuse this paticular instnace?